lib/printer.rb

```diff
@@ -8,16 +8,11 @@ def initialize(col_width)
 
   def print(input)
     @input = input
-    clone_input
     justify_input
     print_input
     clear_job
   end
 
-  def clone_input
-
```

Comment:
Curious to understand what the clone_input method would do, or might be used for, is it needed?